libtransmission/peer-mgr.cc

```diff
@@ -1467,14 +1467,14 @@ namespace get_peers_helpers
 
 [[nodiscard]] bool is_peer_interesting(tr_torrent const* tor, tr_peer_info const& info)
 {
-    if (tor->is_done() && info.is_seed())
+    if (std::empty(info.listen_port()))
     {
         return false;
     }
```

Comment:
The call for `is_peer_interesting` already comes from a peer from `connectable_pool`. How come the `listen_port` be empty? Should we instead simply `TR_ASSERT(!std::empty(info.listen_port()))`?

Reply:
That's true... I added the assert in d2320d2, but I've also kept the `if` block just in case, since it's just a trivial check after all.